tests/test_data_portal.py

```diff
@@ -102,13 +110,25 @@ def make_equity_minute_bar_data(cls):
                 'volume': full(len(dts), 0),
             },
             index=dts))
-        yield 1, pd.concat(dfs)
+        asset1_df = pd.concat(dfs)
+        yield 1, asset1_df
+
+        asset2_df = pd.DataFrame(
+            {
+                'open': 1.0055,
+                'high': 1.0059,
+                'low': 1.0051,
+                'close': 1.0055,
+                'volume': 100,
+            },
+            index=asset1_df.index,
+        )
+        yield 2, asset2_df
 
     @classmethod
     def make_future_minute_bar_data(cls):
-        asset = cls.asset_finder.retrieve_asset(10000)
-        trading_calendar = cls.trading_calendars[asset.exchange]
-        trading_sessions = cls.trading_sessions[asset.exchange]
+        trading_calendar = cls.trading_calendars['us_futures']
```

Comment:
We should be pass the `Future` type here to make it clear that we're getting the primary calendar for futures, and in case we ever want change it.